resources/validations/src/ssp/policy-and-procedure.sch

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sch:schema
+    queryBinding="xslt2"
+    xmlns:sch="http://purl.oclc.org/dsdl/schematron"
+    xmlns:sqf="http://www.schematron-quickfix.com/validator/process">
+
+    <sch:ns
+        prefix="oscal"
+        uri="http://csrc.nist.gov/ns/oscal/1.0" />
+    <sch:ns
+        prefix="fedramp"
+        uri="https://fedramp.gov/ns/oscal" />
+
+    <sch:title>FedRAMP SSP tests</sch:title>
+
+    <sch:pattern>
+
+        <sch:title>A FedRAMP SSP must incorporate one policy document and one procedure document for each of the 17 NIST SP 800-54r4 control
+            families</sch:title>
+
+        <!-- TODO: handle attachments declared by component (see implemented-requirement ac-1 for an example) -->
+
+        <!-- FIXME: XSpec testing malfunctions when the following rule context is constrained to XX-1 control-ids -->
+        <sch:rule
+            context="oscal:implemented-requirement[matches(@control-id, '^[a-z]{2}-1$')]"
```

Comment:
@GaryGapinski I think these assertions are failing due to case-insensitive assumptions on the `XX-1` test contexts.

```suggestion
            context="oscal:implemented-requirement[matches(@control-id, '^[a-zA-Z]{2}-1$')]"
```